google-cloud-spanner/lib/google/cloud/spanner/session.rb

```diff
@@ -150,7 +150,7 @@ def path
         #     specified using a {Fields} object.
         #
         #   Types are optional.
-        # @param [Google::Spanner::V1::TransactionSelector] transaction The
+        # @param [{Google::Cloud::Spanner::V1::TransactionSelector}] transaction The
```

Comment:
Do not use curly braces inside the type spec (square brackets) of a param. In this case, do not use backticks either.